lib/Demangling/Demangler.cpp

```diff
@@ -1458,6 +1458,20 @@ NodePointer Demangler::demangleBoundGenericType() {
   return NTy;
 }
 
+bool Demangle::nodeConsumesGenericArgs(Node *node) {
+  switch (node->getKind()) {
+    case Node::Kind::Variable:
+    case Node::Kind::Subscript:
+    case Node::Kind::ImplicitClosure:
+    case Node::Kind::ExplicitClosure:
+    case Node::Kind::DefaultArgumentInitializer:
+    case Node::Kind::Initializer:
```

Comment:
It seems very easy for this list to go out-of-date. Should we suffer through listing out all of the cases to avoid the `default`?